{
    "cve_id": "CVE-2025-2312",
    "description": "A flaw was found in cifs-utils. When trying to obtain Kerberos credentials, the cifs.upcall program from the cifs-utils package makes an upcall to the wrong namespace in containerized environments. This issue may lead to disclosing sensitive data from the host's Kerberos credentials cache.",
    "cvss_score": 5.9,
    "cwe_id": "CWE-488",
    "cwe_description": "CWE-488",
    "affected_products": [
        {
            "vendor": "cifs-utils",
            "product": "cifs-utils",
            "versions": [
                "0"
            ]
        }
    ]
}